landscape.yml

```diff
@@ -711,50 +760,59 @@ landscape:
+              VSAM (including IAM) files in an online ISPF interface, Eclipse-based user interface, REST API, CLI and through batch commands.
+            homepage_url: >-
+              https://techdocs.broadcom.com/us/en/ca-mainframe-software/devops/ca-brightside/3-0/zowe-cli/available-cli-plug-ins/ca-file-master-plus-plug-in-for-zowe-cli.html
             stock_ticker: AVGO
             logo: file-master-plus.svg
             crunchbase: https://www.crunchbase.com/organization/broadcom
             allow_duplicate_repo: true
           - item:
             name: IBM Z ChatOps
-            description: "IBM Z ChatOps V1.1.2 provides ChatOps capabilities for an IBM Z environment by providing a chatbot that gives users in a supported enterprise\
-              \ chat tool access to information from IBM Z systems management tools.\nIBM Z ChatOps also provides an incident webhook https://your_microservice_server:your_port_number/ibm/bnz/v1/incident\
-              \ to enable other products to send their own incidents, events, and alert data to the chat platform in the required JSON format of common incident data\
-              \ model."
+            description: >-
+              IBM Z ChatOps V1.1.2 provides ChatOps capabilities for an IBM Z environment by providing a chatbot that gives users in a supported enterprise chat
+              tool access to information from IBM Z systems management tools.
+
+              IBM Z ChatOps also provides an incident webhook https://your_microservice_server:your_port_number/ibm/bnz/v1/incident to enable other products to
+              send their own incidents, events, and alert data to the chat platform in the required JSON format of common incident data model.
             homepage_url: https://www.ibm.com/docs/en/z-chatops
-            project:
+            project: null
             logo: IBM Z ChatOps.svg
             twitter: https://twitter.com/ibmz
             crunchbase: https://www.crunchbase.com/organization/ibm
             allow_duplicate_repo: true
           - item:
             name: IBM Z JCL Expert
-            description: IBM Z JCL Expert is a new offering that can accelerate testing and production delivery of JCL streams and reduce production processing errors
-              and costly delays through mass JCL checking.
+            description: >-
+              IBM Z JCL Expert is a new offering that can accelerate testing and production delivery of JCL streams and reduce production processing errors and
+              costly delays through mass JCL checking.
             homepage_url: https://www.ibm.com/products/z-jcl-expert
-            project:
+            project: null
             logo: ibm-z-jcl-expert.svg
             crunchbase: https://www.crunchbase.com/organization/ibm
+            second_path: API Mediation Layer ZOWE V2
```

Comment:
@jmertic 
Wrong second_path! A full path is
`Zowe Conformant / Support Provider API Mediation Layer ZOWE V2`